cms/check_rules_states_by_account

```diff
@@ -15,97 +15,64 @@
-                     .filter(models.ReplicationRule.state == RuleState.STUCK)
-                     .filter(models.ReplicationRule.account == _account))
-
-            stuck = get_count(query)
-            prom_labels = {'account': account, 'state': RuleState.STUCK}
-            prom_labels.update(extra_prom_labels)
-            rules_count_gauge.labels(**prom_labels).set(stuck)
-
-            query = (session.query(models.ReplicationRule.scope)
-                     .filter(models.ReplicationRule.state == RuleState.REPLICATING)
-                     .filter(models.ReplicationRule.account == _account))
-
-            replicating = get_count(query)
-            prom_labels = {'account': account, 'state': RuleState.REPLICATING}
-            prom_labels.update(extra_prom_labels)
-            rules_count_gauge.labels(**prom_labels).set(replicating)
-
-            query = (session.query(models.ReplicationRule.scope)
-                     .filter(models.ReplicationRule.state == RuleState.WAITING_APPROVAL)
-                     .filter(models.ReplicationRule.account == _account))
-
-            wfa = get_count(query)
-            prom_labels = {'account': account, 'state': RuleState.WAITING_APPROVAL}
-            prom_labels.update(extra_prom_labels)
-            rules_count_gauge.labels(**prom_labels).set(wfa)
-
-            # wfa = waiting for approval
-            wfa = get_count(query)
-
-            print('Number of stuck rules by %s: %d' % (account, stuck))
-            print('Number of replicating rules by %s: %d' % (account, replicating))
-            print('Number of waiting for approval rules by by %s: %d' % (account, wfa))
-
-            probe_metrics.gauge(name='rucio.judge.stuck_rules_from.{account}').labels(account=account).set(stuck)
-            probe_metrics.gauge(name='rucio.judge.replicating_rules_from.{account}'
-                                ).labels(account=account).set(replicating)
-            probe_metrics.gauge(name='rucio.judge.wfa_rules_from.{account}').labels(account=account).set(wfa)
-
-            if PROM_SERVERS:
-                for server in PROM_SERVERS:
-                    try:
-                        push_to_gateway(server.strip(), job='check_rules_count_by_state_by_account', registry=registry)
-                    except:
-                        continue
+        accounts = [
+            'transfer_ops', 
+            'wmcore_output', 
+            'wma_prod', 
+            'wmcore_transferor', 
+            'crab_tape_recall'
+        ]
+
+        with PrometheusPusher() as manager: 
+
+            for account in accounts:
+                states = {
+                    "stuck": RuleState.STUCK, 
+                    "replicating": RuleState.REPLICATING, 
+                    "waiting_approval": RuleState.WAITING_APPROVAL
+                }
+
+                for state_name, state in states.items(): 
+                    _account = InternalAccount(account)
```

Comment:
I've never been a fan of prefixing variables like this. In this case account is better as account_name and then _account as account.